client/my-sites/customer-home/main.jsx

```diff
@@ -130,51 +115,32 @@ const Home = ( {
 Home.propTypes = {
 	canUserUseCustomerHome: PropTypes.bool.isRequired,
 	isDev: PropTypes.bool,
-	isStaticHomePage: PropTypes.bool.isRequired,
 	forcedView: PropTypes.string,
 	layout: PropTypes.object,
 	site: PropTypes.object.isRequired,
 	siteId: PropTypes.number.isRequired,
-	trackViewSiteAction: PropTypes.func.isRequired,
 	shuffleViews: PropTypes.bool,
 };
 
 const mapStateToProps = ( state ) => {
 	const siteId = getSelectedSiteId( state );
-	const isClassicEditor = getSelectedEditor( state, siteId ) === 'classic';
 	const layout = getHomeLayout( state, siteId );
 
 	return {
 		site: getSelectedSite( state ),
 		siteId,
 		canUserUseCustomerHome: canCurrentUserUseCustomerHome( state, siteId ),
-		isStaticHomePage:
-			! isClassicEditor && 'page' === getSiteOption( state, siteId, 'show_on_front' ),
 		layout,
 	};
 };
 
-const trackViewSiteAction = ( isStaticHomePage ) =>
-	composeAnalytics(
-		recordTracksEvent( 'calypso_customer_home_my_site_view_site_click', {
-			is_static_home_page: isStaticHomePage,
-		} ),
-		bumpStat( 'calypso_customer_home', 'my_site_view_site' )
-	);
-
-const mapDispatchToProps = {
-	trackViewSiteAction,
-};
-
 const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
-	const { isStaticHomePage } = stateProps;
 	return {
 		...ownProps,
 		...stateProps,
-		trackViewSiteAction: () => dispatchProps.trackViewSiteAction( isStaticHomePage ),
 	};
 };
 
-const connectHome = connect( mapStateToProps, mapDispatchToProps, mergeProps );
+const connectHome = connect( mapStateToProps, null, mergeProps );
```

Comment:
With `trackViewSiteAction` going away I think we can actually get rid of all this prop merging code! Looks like it was only there so we could forward the `isStaticHomePage` prop to the `trackViewSiteAction()` function.

```suggestion
const connectHome = connect( mapStateToProps );
```

And then we can delete `mergeProps` entirely

Reply:
Nice spot - thank you! Should be removed now. :)